cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -134,10 +134,35 @@ describe(["@tier1"], "Application assessment and review tests", () => {
 
         application.delete();
         cy.wait(2 * SEC);
+
+        AssessmentQuestionnaire.delete(cloudNative);
+        cy.wait(2 * SEC);
+    });
+
+    it("Perform application review during assessment", function () {
```

Comment:
Could you please add the Polarion TC ?

Reply:
I didn't see other test cases having the polarion link added, it's not organized to so only for some of the test cases and leaving others, wdyt?

Reply:
You could add the Polarion TC ID for these: 
1) Perform application review during assessment
2) Perform assessment using multiple questionnaires